api/config/module/passthrough/v1/passthrough.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package clutch.config.module.passthrough.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/config/module/passthrough/v1;passthroughv1";
+
+import "validate/validate.proto";
+
+message Config {
+  repeated Service services = 1;
+}
+
+message Service {
+  // An identifier to reference the service
+  string name = 1 [ (validate.rules).string.min_len = 1 ];
+  // The host URL to communicate with
+  string host = 2 [ (validate.rules).string.min_len = 1 ];
+  // Paths that are allowed to be invoked
+  repeated string allowed_paths = 3 [ (validate.rules).repeated = {min_items : 1} ];
+  // HTTP methods that are allowed to be invoked
+  repeated string allowed_methods = 4 [ (validate.rules).repeated = {min_items : 1} ];
+  // Additional headers can be set such as auth
```

Comment:
these should probably be combined into a message so I can allowlist the combination of path/method rather than each separately.

e.g. 
DELETE /alerts/escalation_policies
GET /alerts
POST /alerts